infra/cifuzz/fuzz_target.py

```diff
@@ -110,6 +113,7 @@ def fuzz(self):
     ]
     run_fuzzer_command = 'run_fuzzer {fuzz_target} {options}'.format(
         fuzz_target=self.target_name, options=LIBFUZZER_OPTIONS)
+    run_fuzzer_command += ' -max_total_time=' + str(self.duration)
```

Comment:
let's do `options = LIBFUZZER_OPTIONS + ' -max_total_time=' + str(self.duration)` on line 113, and then use `options=options` in line 115, that way all the options we've used will be inside a single variable

Reply:
also, is `duration` guaranteed to be an integer?

Reply:
Updated.